watch-library/shared/watch/watch_common_display.c

```diff
@@ -57,6 +57,7 @@ void watch_display_character(uint8_t character, uint8_t position) {
             else if (character == 'c') character = 'C'; // C needs to be uppercase
             else if (character == 'J') character = 'j'; // same
             else if (character == 'v' || character == 'V' || character == 'U' || character == 'W' || character == 'w') character = 'u'; // bottom segment duplicated, so show in top half
+            else if (character == 't' || character == 'T') character = '+'; // bottom and top segment duplicated, so show as ┣╸
```

Comment:
On the classic display in pos 4 & 6 the chars `t` and `T` would display as:
<img width="243" height="212" alt="Screenshot 2025-07-19 at 10 59 29 PM" src="https://github.com/user-attachments/assets/59000fb3-be9e-4aa6-9cf3-67f10ce9ce18" />

This change now makes them display as:
<img width="245" height="208" alt="Screenshot 2025-07-19 at 11 09 33 PM" src="https://github.com/user-attachments/assets/9110de8d-54b6-469d-9097-656601665c2c" />